### ISSUE TITLE
SCMS - Social Curreny Metric System

### ISSUE THREAD

**Issue by germonprez** [chaoss/wg-value#116]
Hi @ElizabethN @GeorgLink @sgoggins @samanthavenialogan @mbbroberg 

I watched the video from Friday's SCMS meeting. Thanks for recording this. At the end of the video I newly learned that SCMS has been awarded(?) applied to(?) as a trademark. I think this is an important issue. My first reaction is that I'm not sure that having a trademarked name as a metric is a great idea. Perhaps SCMS stays uniquely with the work you are doing with Socially Constructued @samanthavenialogan. 

In the CHAOSS project, we could perhaps start working with a rebranded and scaled back metric as has been contributed and worked on to date (as was also discussed in the video). 

What do people think about this? Video here: https://www.youtube.com/watch?v=evpFtwNkBwM

**Comment by GeorgLink** [comment]
Friday was the first time I learned about the possible trademark as well. Venia and Dylan can speak to the status on that.

I agree that as a CHAOSS project we should avoid infringing on any intellectual property. I’ll reach out to the LF and ask for guidance on how we can deal with situations involving trademarked metrics.

**Comment by samanthavenialogan** [comment]
I already spoke with Georg and Dylan on this as well.

This is a non-issue because I was mistaken on the trademark because I
rememberd Dylan and I discussing it and then us submitting for it.  After
doing research we had an application filled out all the way up to pay for
it then it looks like we didn't.  I had remembered the initial status
incorrectly and this is my fault.

The trademark itself was also only to cover the logo and not the system
itself so even if it was present, on our end at least, we wouldn't see any
violation there on our end.

Sorry, for the confusion and inconvenience there. It's my fault for
remembering incorrectly.

Samantha Venia Logan
Co-Founder of SociallyConstructed.Online
307-274-5516  | samantha@SociallyConstructed.Online
6715 Autumn Ridge Dr. Unit 2, Fort Collins CO., 80525

**Comment by GeorgLink** [comment]
Thanks for clarifying. I'm glad there is no conflict.

**Comment by germonprez** [comment]
Yes, thanks for clarifying but I think if this is on your agenda, it would make sense to address this now.

**Comment by samanthavenialogan** [comment]
Yeah we are discussing it internally before we bring more to you on it when
a decision is ultimately made.  It might be good to explore the potential
issues if we do move forward so we can figure that in to our decision.
Would you be willing to just inquire softly with the linux foundation so we
can get some advice on that? I must admit I'm VERY bad when it comes to
legal stuff.

Co-Founder of SociallyConstructed.Online
307-274-5516  | samantha@SociallyConstructed.Online
6715 Autumn Ridge Dr. Unit 2, Fort Collins CO., 80525

**Comment by GeorgLink** [comment]
Yes, I'm asking for guidance from the Linux Foundation. I'll update you
when I hear back.

**Comment by GeorgLink** [comment]
Hi everyone, As discussed at our last meeting, I opened a Doodle poll to see when the most of us would be available for a bi-weekly SCMS coordination meeting. This is outside of the regular Value WG calls because Venia and Dylan cannot make those.

https://doodle.com/poll/f6iid9bi6gnkcs5h

The poll shows next week, but please answer it as if this was for future weeks.

/cc @samanthavenialogan @sgoggins @germonprez @ElizabethN @klumb @mbbroberg ...

**Comment by ElizabethN** [comment]
Closing this, as this has been moved to the Metrics Model Working Group.

**Comment by samanthavenialogan** [comment]
Hey is there any need for me to act here? I feel like just starting with
the metrics from scratch makes the most sense but trying to do this and
build the core business has been difficult.

Samantha Venia Logan
Online Community Architect & Founder
SociallyConstructed.Online
970-694-8339  | ***@***.***
***@***.***>
6715 Autumn Ridge Dr. Unit 2, Fort Collins CO., 80525
